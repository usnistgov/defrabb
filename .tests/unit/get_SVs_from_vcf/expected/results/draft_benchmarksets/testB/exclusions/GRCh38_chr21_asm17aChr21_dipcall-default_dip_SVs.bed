chr21	9586587	9586601
chr21	9676569	9676572
chr21	9719626	9720115
chr21	9719628	9720115
chr21	9768004	9768641
chr21	9801028	9801042
chr21	9882313	9882327
chr21	9981074	9981321
chr21	9997167	9997485
chr21	10009887	10010215
chr21	10014851	10015013
chr21	10129121	10129168
