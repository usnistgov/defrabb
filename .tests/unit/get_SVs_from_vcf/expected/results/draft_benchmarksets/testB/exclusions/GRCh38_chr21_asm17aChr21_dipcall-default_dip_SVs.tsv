chr21	9586586	G	GTGATGATGATGGTGAGGATGGTGATGGTGAAGGTGGAGGTGGTGATAATGAAGTGATAGTTATGTTGAGGATGATGAAGGTGATGATGATGGTGAGAATGGCAAAGGTAAACACGGTGATGGTGAGAATGGTGATGGCAGTGATTGTGATGGTGATGATAAGGATGGTGATGGTGAAGGTGGTGATGATAATAGTGATATTTATGGTGAGGATAATGGTGACGATGAGGATGATGATATTGAGAATGGTGATAGTGATGATGATGGTGAGGATGGTGATGGTGAAGGTGGAGGTGGTGATAATGAAGTGATAGTTATGTTGAGGATGATGAAGGTGATGATGATGGTGAGAATGGCAAAGGTAAACACGGTGATGGTGAGAATGGTGATGGCAGTGATTGTGATGGTGATGATAAGGATGGTGATGGTGAAGGTGGTGATGATAATAGTGATATTTATGGTGAGGATAATGGTGACGATGAGGATGATGATATTGAGAATGGTGATA	chr21:9586587-9586601	SIMPLEINS
chr21	9591066	C	CATAGACACAAGGTTTCACTATATTTAGACACAAGGTTTCACTAT	chr21:9591067-9591070	SIMPLEINS
chr21	9596222	G	GGCTGATCCTTGGAGAAGTTCTGGTTGATTTGGGA	chr21:9596221-9596229	DUP
chr21	9596382	T	TTGATTTGGGAGCTGATCCTTGGAGAAGTTCCGGC	chr21:9596381-9596413	DUP
chr21	9676568	T	TTACAACCCTAACCCTGACTTCCCTAATTCCCCCCATCCTTACCACCCTCGTTAACCCTAACAAAAAAAACTCATACCCCCATTATGTAAAATCCATTGTCGCATCCACCTTTATTATCAGTCTCTTCCCCACAACAATATTCATGTGCCTAGACCAAGAAGTTATTATCTCGAACTGACACTGAGCCACAACCCAAACAACCCAGCTCTCCCTAAGCTTCAAACTAGACTACTTCTCCATAATATTCATCCCTGTAGCATTGTTCGTTACATGGTCCATCATAGAATTCTCACTGTGATATATAAACTCAGACCCAAACATTAATCAATTCTTCAAATATCTACTCATTTTCCTAATTACCATACTAATCTTAGTTACCGCTAACAACCTATTCCAACTGTTCATCGGCTGAGAGGGCGTAGGAATTATATCCTTCTTGCTCATCAGTTGATGATACGCCCGAGCAGATGCCAACACAGCAGCCATTCAAGCAATCCTATACAACCGTATCGGCGATATCGGTTTCATCCTCGCCTTAGCATGATTTATCCTACACTCCAACTCATGAGACCCACAACAAATAGCCCTTCTAAACGCTAATCCAAGCCTCACCCCACTACTAGGCCTCCTCCTAGCAGCAGCAGGCAAATCAGCCCAATTAGGTCTCCACCCCTGACTCCCCTCAGCCATAGAAGGCCCCACCCCGGTCTCAGCCCTACTCCACTCAAGCACTATAGTCGTAGCAGGAGTCTTCTTACTCATCCGCTTCCACCCCCTAGCAGAAAATAGCCCACTAATCCAAACTCTAACACTATGCTTAGGCGCTATCACCACTCTGTTCGCAGCAGTCTGCGCCCTTACACAAAATGAC	chr21:9676569-9676572	SIMPLEINS
chr21	9705586	G	GTTCACCCCGCCGCGCTCCCGCCTTCGTCAGCGTCTC	chr21:9705585-9705612	DUP
chr21	9705650	TAGCGTCTGTTCACCCCGCCCCGCTCCCGCCTTCGTC	T	chr21:9705649-9705730	CONTRAC
chr21	9719626	GGGTTCTACTCGGTGACAACGGAACCATTCCGTTTCATGTGATTCATTATAGAGTTCCTCCAACATAGC	G	chr21:9719626-9720115	CONTRAC
chr21	9719637	G	GGGTGACAAGGAACCATTCCGTTTCATGTGATTCATTTATAGAGTTCCTCTAATATAGCACGTTCTACTCATGACAACGGAACCATTCCGTTTCATGTGATTCATTTATAGAGTTCCTCTAATATAGCACGTTCTACTCA	chr21:9719628-9720115	DUP
chr21	9722737	GGTTGTAGTGAGCCGAGATCATGCCACTGCACTCC	G	chr21:9722736-9722772	SIMPLEDEL
chr21	9761944	T	TGTGTGTGTGTGTGTGTTGTGTGTGTGTGTGTG	chr21:9761943-9761961	DUP
chr21	9765489	A	AGTGTAGTGAATGATGGGACAGT	chr21:9765488-9765491	DUP
chr21	9768007	G	GCGGGGCTGAGGGCTCCTCCA	chr21:9768006-9768020	DUP
chr21	9768070	A	AGAGTTCCTCTGCTGAGCTTGCGCCCTCACCCTCCCTGAGCTTCTGAAGGGCT	chr21:9768004-9768641	DUP
chr21	9790175	CCTCTCTCCAGGCTCACACATTGAAGAGA	C	chr21:9790174-9790204	SIMPLEDEL
chr21	9794803	A	AATATATATATATATATATACAC	chr21:9794802-9794804	DUP
chr21	9800660	CAGGCTGGTCTTGAGCTCCTGACCTCAAGTGA	C	chr21:9800659-9800692	SIMPLEDEL
chr21	9801029	G	GGGGTGGTGTAGGAAAGGGGTGGTGTAGGAAAGGGTGGTGTAGGGAAGGGTTTGGTGTAGGAAAGGGGTGGTGTAGGAAAGGGTGGTGTAGGGAA	chr21:9801028-9801042	DUP
chr21	9801032	G	GTGGTGTAGGAAAGGGGTGGTGTAGGAAAGGGTGGTGTAGGGAAGGGT	chr21:9801031-9801042	DUP
chr21	9858103	CATATATATATATATATATATATATATATATA	C	chr21:9858102-9858135	SIMPLEDEL
chr21	9874561	AAACTCCATCTCTCACACACACACAC	A	chr21:9874560-9874588	SIMPLEDEL
chr21	9882312	T	TAAACAGTACCCAGGTCGGCCGGGCGCGGTGGCTCACGCCTGTAATCCCAGCACTTTGGGAGGCCGAGGCGGGTGGATCATGAGGTCAGGAGATCGAGACCATCCTGGCTAACAAGGTGAAACCCCGTCTCTACTAAAAATACAAAAAATTAGCCGGGCGCGGTGGCGGGCGCCTGTAGTCCCAGCTACTCGGGAGGCTGAGGCAGGAGAATGGCGTGAACCCGGGAAGCGGAGCTTGCAGTGAGCCGAGATTGCGCCACTGCAGTCCGCAGTCCGACCTGGGCGACAGAGCGAGACTCCGTCTCAAAAAAAAAAAAAAAAAAAAAAAAAA	chr21:9882313-9882327	SIMPLEINS
chr21	9981099	TTTCTTTCTTTTTCTTTCTTTCTTTCTTTCTTTCTTTCTTTCTTTCTTTCTTTTTCTTTCTTTCTTTCTTTCTTTCTTTCTTTCTTTCTTTCTTTCCTTCCTTCCTTC	T	chr21:9981074-9981321	CONTRAC
chr21	9981207	G	GTCTTTCTTTCTTTCTTTCTTT	chr21:9981206-9981209	DUP
chr21	9997166	TAGATAATGCAGTAAGGGCCGGGCGCGGTGGCTCACGCCTGTAATCCCAGCACTTTGGGAGGCCGAGATGGGCAGATCACGAGGTCAGGAGATCGAGACCATCCTGGCTAACACGGTGAAACCCCGTCTCTACTAAAAATACAAAAATTAGCCGGGCATGGTGGCGCACGCCTGTAGTCCCAGCTACACGGGAGGCTGAGGCAGGAGAATGGCGTGAACCCGGGAGGCGGAGCTTGCAGTGAGTCGAGATCGCGCCACTGCACTCCAGCCTGGGCGACAGAGCGAAACTCCGTCTCAAAAAAAAAAAAAAAAAAAAAAA	T	chr21:9997167-9997485	SIMPLEDEL
chr21	10009886	TATAAAATATATATTTTTTTTTTTTTTTTTTTTTTTTTTTTTTGAGACGGAGTCTCGCTCTGTCGCCCAGGCCGGACTGCGGACTGCAGTGGCGCAATCTCGGCTCACTGCAAGCTCCGCCTCCCGGGTTCACGCCATTCTCCTGCCTCAGCCTCCCGAGTAGCTGGGACTACAGGCGCCCGCCACCGCGCCCGGCTAATTTTTTGTATTTTTAGTAGAGACGGGGTTTCACCGTGTTAGCCAGGATGGTCTCGATCTCCTGACCTCGTGATCCACCCGCCTCGGCCTCCCAAAGTGCTGGGATTACAGGCGTGAGCCACCGCGCCCGGC	T	chr21:10009887-10010215	SIMPLEDEL
chr21	10014850	TAGATCGCCGCCGGGAAAAAAGGCGGGAGAAGCCCCGGCAGGTTTGAAGCTGCTTCTTCGAATTTGCAATTCAATATGAAAATCACCTCAGAGCTGGTAAAAAGAGGCTTAACCCCTGTCTTTAGATTTACAGTCCAATGCTTCACTCAGCCATTTTACCTCAC	T	chr21:10014851-10015013	SIMPLEDEL
chr21	10129134	T	TCTGATGTAAACCTCTAGTTCAGTATCCACCTGGGCCTGGTCTTCGCTTGGGGCCTGCTGTCTACCTGGGCCTTGCTGTCAACCTAGGGCCGATGTAAACCTCTAGTTCAGTATCCACCTGGGCCTGGTCTTCGCTTGGGGCCTGCTGTCTACCTGGGCCTTGCTGTCAACCTAGGG	chr21:10129121-10129168	SIMPLEINS
chr21	10140920	ATGTGTGTATATATTACATATATACG	A	chr21:10140919-10141005	CONTRAC
